CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 2.14.0
+
+### Changed
+
+* bumps `application/vnd.elife.article-vor+json` from `version=6` to `version=7` in the API
+    - version 7 article-vor responses may contain elifeAssessment, recommendationsForAuthors and publicReviews
+
+### Removed
+
+* removed support for deprecated `application/vnd.elife.article-vor+json;version=5`
```

Comment:
you've also removed this for vor articles v6:

```
    anyOf:
      - dependencies:
            authorResponse: [decisionLetter]
      - dependencies:
            authorResponse: [editorEvaluation]
```

Reply:
This is a mistake. I will reinstate.

Reply:
> you've also removed this for vor articles v6:

ack! the mistake is mine, I made a typo, they've been removed in **v7**, they're still present in **v6**. Still need a mention under **Removed** though.

Reply:
Made alterations to CHANELOG. If you're still not happy with it please just submit a followup commit. I'm sure it will be fine but can review if you want me to look it over.